packages/documentation-site/patternfly-docs/content/design-guidelines/patterns/actions/actions.md

```diff
@@ -0,0 +1,88 @@
+---
+id: Actions
+section: patterns
+---
+
+An **action** is any process that a user can trigger by clicking or selecting a linked component. Common actions include adding, deleting, editing, filtering, and submitting, for example. In PatternFly, most actions are shown in the UI as action buttons, action items (in menus), and action cards. These components are incorporated into the design of other components, such as a toolbar with action buttons, a dropdown menu with action items, and so on.
+
+## Elements 
+
+The following image of a card view demonstrates some of the more common uses of actions that you will find in a UI, but does not cover all scenarios. 
+
+![A card view, where the toolbar action buttons, actionable card, ellipsis icon, and action menu item are marked with annotations.](./img/actions-pattern.png)
```

Comment:
Would remove the icon in front of the menu item "Delete card" because it's not a common thing to do for menu items - don't want to confuse people!